pkg/cmd/build/build.go

```diff
@@ -104,6 +104,9 @@ func (ob *OktetoBuilder) Run(ctx context.Context, buildOptions *types.BuildOptio
 	}
 
 	switch {
+	// When depot is available we only go to depot if it's not a deploy or a destroy.
```

Comment:
@maroshii be aware of this for the `okteto test`. It should not go to Depot either